DependencyInjection/Compiler/MiddlewaresPass.php

```diff
@@ -16,6 +17,11 @@ final class MiddlewaresPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
+        // Middlewares introduced in `doctrine/dbal:3.0.0`
+        if (! interface_exists(Middleware::class)) {
```

Comment:
So as I see it this pass shouldn't even be registered using dbal 2?

https://github.com/doctrine/DoctrineBundle/blob/2.6.x/DoctrineBundle.php#L68

Reply:
Yes, it should not be registered.

Then I don't know the reason for the error from #1485.

Reply:
Yes, it should not be registered.

Then I don't know the reason for the error from #1485.
:(